content/rendering-with-batch/monitor/optional_fis.md

```diff
@@ -0,0 +1,198 @@
+---
```

Comment:
We always open a page with a descriptive message of what they will achieve by running the subsequent commands, and the why to that.

Reply:
For every AWS CLI api call that is make, include a reference in the documentation for the readers to expand on it if they want.